db-utils.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+const pg = require("pg");
+
+// Use PG* env vars to configure client.
+const client = new pg.Client();
+
+const DBUtils = {
+  async setupDatabase() {
+    try {
+      await client.connect();
+      await client.query("DROP TABLE IF EXISTS users;");
```

Comment:
Should we add any magical checks to make sure this isn't run in `process.env.NODE_ENV=production`?
I tried doing a quick Ctrl+F on this page and only saw the one reference in tests/test-subscribers.js, but something about `DROP TABLE` makes me super paranoid that something gets nuked during when deploying or scaling.